main.go

```diff
@@ -37,6 +37,11 @@ const bwDisconnectedPrefix = "𐄂"
 const bwErrorPrefix = "!"
 const bwInfoPrefix = "·"
 
+type ErrorMessage struct {
```

Comment:
Maybe, I would rename this struct to `LogMessage`, because this structure can hold also warning and debug messages.